src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityStrafe.kt

```diff
@@ -44,6 +58,17 @@ internal object VelocityStrafe : VelocityMode("Strafe") {
 
         // Check if this is a regular velocity update
         if ((packet is EntityVelocityUpdateS2CPacket && packet.entityId == player.id) || packet is ExplosionS2CPacket) {
+            val target = world.findEnemy(0f..OnlyFacing.range)
+            val isFacingEnemy = facingEnemy(
+                target!!,
+                OnlyFacing.range.toDouble(),
+                RotationManager.currentRotation ?: player.rotation
+            )
```

Comment:
This should be moved to the game tick, as we are dealing with packets running in a different thread causing concurrent modification.